### ISSUE TITLE
Let's select a paper for our paper reading/discussion session on June 20, 2020

### ISSUE THREAD

**Issue by omarsar** [dair-ai/nlp-paper-discussions#4]
Suggest a paper you would like us to discuss during our weekly paper reading discussion. It can be a paper from RL, Computer Vision, NLP, or any ML related paper. 

You can vote on a suggested paper by using the 👍 emoji. I will close the issue in one or two days and select the paper with the most votes. Then I will make the announcement at the beginning of the week. Thanks.

**Comment by ajan1019** [comment]
On the Measure of Intelligence by Francois Chollet.

https://arxiv.org/pdf/1911.01547.pdf

**Comment by cahya-wirawan** [comment]
Linformer: Self-Attention with Linear Complexity
From Facebook
https://arxiv.org/abs/2006.04768

**Comment by eyebies** [comment]
[Learning To Classify Images Without Labels](https://arxiv.org/abs/2005.12320)

**Comment by jithendray** [comment]
[SeqGAN: Sequence Generative Adversarial Nets with Policy Gradient](https://arxiv.org/abs/1609.05473)

**Comment by morganmcg1** [comment]
[VirTex: Learning Visual Representations from Textual Annotations](https://arxiv.org/pdf/2006.06666.pdf)

Pretrain: CNN+Transformer from scratch on COCO Captions.

Transfer CNN: Results on 6 vision tasks match/exceed ImageNet pretraining (10x size wrt COCO)

*NLP, transformers, vision + imagenet performance, whats not to love :-D*

**Comment by sai-prasanna** [comment]
[Deep Dominance - How to Properly Compare Deep Neural Models ](https://www.aclweb.org/anthology/P19-1266/)

> Comparing between Deep Neural Network (DNN) models based on their performance on unseen data is crucial for the progress of the NLP field. However, these models have a large number of hyper-parameters and, being non-convex, their convergence point depends on the random values chosen at initialization and during training. Proper DNN comparison hence requires a comparison between their empirical score distributions on unseen data, rather than between single evaluation scores as is standard for more simple, convex models. In this paper, we propose to adapt to this problem a recently proposed test for the Almost Stochastic Dominance relation between two distributions. We define the criteria for a high quality comparison method between DNNs, and show, both theoretically and through analysis of extensive experimental results with leading DNN models for sequence tagging tasks, that the proposed test meets all criteria while previously proposed methods fail to do so. We hope the test we propose here will set a new working practice in the NLP community.

**Comment by omarsar** [comment]
This was a close one. We have a draw. To diversify things a bit, I would like to select the paper suggested by @morganmcg1: 

VirTex: Learning Visual Representations from Textual Annotations

Thanks again for all the help and looking forward to another great paper reading and discussion session.
